pkgs/servers/monitoring/zabbix/proxy.nix

```diff
@@ -29,6 +30,23 @@ assert sqliteSupport -> !mysqlSupport && !postgresqlSupport;
 
 let
   inherit (lib) optional optionalString;
+
+  fake_pg_config = buildPackages.writeShellScript "pg_config" ''
+    if [[ "$1" == "--version" ]]; then
+      $PKG_CONFIG libpq --modversion
+    else
+      $PKG_CONFIG libpq --variable="''${1//--/}"
+    fi
+  '';
```

Comment:
pg_config is in postgresql.dev, so it _should_ be executable on the build platform. how hard is it to fix that in the postgresql expression instead of here? same question for `libmysqlclient` below.

Reply:
`.pg_config-wrapped` is a binary, and so is `mysql_config`